### DISCUSSION TITLE
Configurable livenessProbe and readinessProbe

### DISCUSSION THREAD

**Discussion by Kienan6** [directus-labs/helm-chart#28] [Ideas]
Currently the chart defines the livenessProbe and readinessProbe without any option for configurability. For my particular setup it would be great to adjust some of the parameters associated with these checks. I'd imagine that others could use this feature as its available on other popular charts.

**Comment by mykhailo-sindieiev** [comment]
Sounds cool. Fell free to open a freestyle PR or I will include those features into the next release

**Reply by Kienan6** [reply]
Great, here you go. https://github.com/directus-labs/helm-chart/pull/39

**Reply by mykhailo-sindieiev** [reply]
Alright, thank you. I'll try to deploy it to the cluster tomorrow and merge

**Comment by mykhailo-sindieiev** [comment]
Merged. And, I made a new release with changes
